File: tests/java/tests-java-gradle-junit5-allure3-selenide/src/test/java/tests/integration/RegisterFormTests.java
package tests.integration;

import tests.TestBase;
import annotations.Layer;
import io.qameta.allure.Epic;
import io.qameta.allure.Feature;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Tag;
import org.junit.jupiter.api.Test;
import pages.RegisterPage;

@Layer("integration")
@Epic("Authentication")
@Feature("Register form")
@DisplayName("Register form mount")
class RegisterFormTests extends TestBase {

    private final RegisterPage registerPage = new RegisterPage();

    @Test
    @Tag("mount")
    @DisplayName("Register form fields and submit are visible")
    void registerFormIsMounted() {
        registerPage.openPage()
                .shouldShowRegisterForm()
                .shouldHaveFormTitle("Register");
    }
}

File: tests/csharp/tests-csharp-xunit-api_request-playwright/Tests/Ui/RegisterFormTests.cs
using Allure.Net.Commons;
using Allure.Net.Commons.Attributes;
using Tests;

namespace Tests.Ui;

[AllureLabel("layer", "ui")]
[AllureEpic("Authentication")]
[AllureFeature("Register form")]
[AllureSeverity(SeverityLevel.normal)]
[AllureSuite("Register form mount")]
public sealed class RegisterFormTests : TestBase
{
    [Trait("TestCategory", "ui")]
    [Trait("TestCategory", "mock")]
    [Fact(DisplayName = "Register form fields and submit are visible")]
    public void RegisterFormIsMounted()
    {
        RegisterPage.OpenPage()
            .ShouldShowRegisterForm()
            .ShouldHaveFormTitle("Register");
    }
}
